<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="side_hjem">
       <widget class="QPushButton" name="neste_1">
        <property name="geometry">
         <rect>
          <x>1180</x>
          <y>700</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Neste</string>
        </property>
       </widget>
       <widget class="QLabel" name="logo_tronrud">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>310</width>
          <height>78</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="logo_pvr">
        <property name="geometry">
         <rect>
          <x>1080</x>
          <y>0</y>
          <width>178</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>280</y>
          <width>711</width>
          <height>154</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="bruker_pass">
         <item row="2" column="1">
          <widget class="QTextEdit" name="textEdit_2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="passord">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Passord</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="brukernavn">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Brukernavn</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="side_send_motta">
       <widget class="QPushButton" name="x_1_main">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="send">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>260</y>
          <width>201</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="motta">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>260</y>
          <width>201</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Motta</string>
        </property>
       </widget>
       <widget class="QLabel" name="logo_tepvr">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>0</y>
          <width>481</width>
          <height>94</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
